packages/mimecast/data_stream/dlp_logs/manifest.yml

```
@@ -14,6 +14,14 @@ streams:
         required: true
         show_user: false
         default: 5m
+      - name: alerting
+        type: text
+        title: Altert Actions
```

```suggestion
        title: Alert Actions
```
Other places as well.

---

Thanks